_gsocblogs/2024/blog_ATLCompression_IshanDarji.md

```diff
@@ -0,0 +1,120 @@
+# Lossless compression of raw data for the ATLAS experiment at CERN
+
+### Abstract
+The goal of this project is to study the performance and effectiveness of various compression algorithms, specifically on ATLAS RAW data. The [ATLAS experiment](https://arxiv.org/abs/2404.06335) produces extremely large amounts of data, and it is only expected to increase with future planned upgrades within the LHC. Prior studies into compression of the data have shown that due to the highly redundant nature of the generated data, lossless data compression algorithms are extremely effective in reducing the binary size of ATLAS data. Here, we would like to find an algorithm that has a good balance of compression time, and compressed binary size, with the compression ratio being of much higher priority than the compression time due to the size of the data being compressed. One additional requirement is that the data must be held in its compressed state for an indefinite period until it is losslessly compressed at some point in the future. Ultimately, the research conducted in this project will be able to guide important decisions regarding the compression scheme implemented in the ATLAS experiment. 
```

Comment:
I suggest rephrasing
> until it is losslessly compressed at some point in the future

for clarity. The thing is that in science reproducibility is a must. So we need to be able to read this data in future.